builtin/settingtypes.txt

```diff
@@ -1682,8 +1682,8 @@ mgfractal_dungeon_ymax (Dungeon maximum Y) int 31000
 #    8 = 4D "Variation" julia set.
 #    9 = 3D "Mandelbrot/Mandelbar" mandelbrot set.
 #    10 = 3D "Mandelbrot/Mandelbar" julia set.
-#    11 = 3D "Christmas Tree" mandelbrot set.
-#    12 = 3D "Christmas Tree" julia set.
+#    11 = 3D "Yule Tree" mandelbrot set.
+#    12 = 3D "Yule Tree" julia set.
```

Comment:
Incorrect, this fractal is known specifically as the 'Christmas Tree'.
http://www.bugman123.com/Hypercomplex/index.html